macros/src/lib.rs

```diff
@@ -512,6 +526,7 @@ fn log(level: Level, log: FormatArgs) -> TokenStream2 {
 
     let sym = mksym(&ls, level.as_str(), true);
     let logging_enabled = cfg_if_logging_enabled(level);
+    let logging_disabled = cfg_if_logging_disabled(level);
     quote!({
         #[#logging_enabled] {
```

Comment:
If you change this to

```suggestion
        #[cfg(#logging_enabled)] {
```

and change the code above to not include `cfg()`, you don't need 2 separate functions and `quote!` invocations, that should be a bit easier to read

Reply:
That's smart 🥲